### PR TITLE
Feature tca 24 laptop bags

### PR DESCRIPTION
**Pull request by miczaj90**

### CODE REVIEW

File: ITCrowdProject/src/main/java/pl/infoshare/pages/LaptopBagsCataloguePage.java [hunk 1]
```diff
@@ -4,27 +4,33 @@
...
 import pl.infoshare.elements.Button;
 import pl.infoshare.pages.Sections.ShippingCart;
+import pl.infoshare.elements.Label;
+
 
 
 public class LaptopBagsCataloguePage extends BasePage {
 
     private Button addToCart;
+    private By shippingCartLink= By.cssSelector("#miniCartSummary a span");
```
Comment (Mateusz-Zielinski) [inline ITCrowdProject/src/main/java/pl/infoshare/pages/LaptopBagsCataloguePage.java]: By.Id

File: ITCrowdProject/src/main/java/pl/infoshare/pages/FooterPage.java [hunk 2]
```diff
@@ -27,4 +28,7 @@ public void clickHandbagsButtonOnFooter(){
     }
 
 
+    public void clickLaptopBagsButtonOnFuter(){
```
Comment (Mateusz-Zielinski) [inline ITCrowdProject/src/main/java/pl/infoshare/pages/FooterPage.java]: clickLaptopBagsButtonOn**Futer** zamień na clickLaptopBagsButtonOnFooter

File: ITCrowdProject/src/main/java/pl/infoshare/pages/LaptopBagsCataloguePage.java [hunk 3]
```diff
@@ -4,27 +4,33 @@
...
+    private Button shoppingCart;
     private By chackoutLink;
     private Button checkout;
     private By bag;
+    private By laptopBagsTekst = By.xpath("//h2[@class='shop-banner-title lead']");
```
Comment (katarzynaboguslawska) [inline ITCrowdProject/src/main/java/pl/infoshare/pages/LaptopBagsCataloguePage.java]: laptopBagsText

File: ITCrowdProject/src/main/java/pl/infoshare/pages/LaptopBagsCataloguePage.java [hunk 4]
```diff
@@ -40,4 +46,10 @@ public void getRandomBag(String name){
         bag = By.cssSelector("#productsContainer [item-name='"+name+"'] div.store-btn-addtocart");
         this.addToCart = new Button(this.driver, this.bag);
     }
+
+    public String laptopBagTekst(){
```
Comment (katarzynaboguslawska) [inline ITCrowdProject/src/main/java/pl/infoshare/pages/LaptopBagsCataloguePage.java]: Text

**Pull request merged**